TAC/Meetings/2022/2022-04-07/CCC_TAC_Minutes-2022-04-07.md

```diff
@@ -41,89 +42,117 @@
 
 ### Roll Call / Attendance
 
 #### Voting members
 
-<!-- * Dan Middleton (Intel)
+* Simon Johnson (Intel)
 * Dave Thaler (Microsoft)
-* Eric Northup (Meta)
 * Lily Sturman (Red Hat)
-* Zhipeng (Howard) Huang (Huawei)
-* Thomas Fossati (Arm) -->
+* Thomas Fossati (ARM)
+* Eric Northup (Meta)
 
-#### Other attendees
+#### Attendees
 
-<!--* Ben Fischer (Red Hat)
-* Brian Warner (Linux Foundation)
 * Eric Voit (Cisco)
-* Helen Lau (Linux Foundation)
-* Henk Birkholz ()
-* Kurt Taylor (Linux Foundation)
-* Naveen Cherukuri (Nvidia)
-* Mike Bursell (Profian)
 * Nick Vidal (Profian)
 * Penglin Yang (China Mobile)
-* Sneha Damle (R3)
-* Simon Johnson (Intel)
 * Steve Van Lare (Anjuna)
-* Xinxin Fan (IoTeX) -->
+* Ben Fischer (Red Hat)
+* Mark Novak (JP Morgan Chase)
+* Naveen Cherukuri (NVIDIA)
+* Jethro Beekman (Fortanix)
+* Henk Birkholz (Fraunhofer-Gesellschaft)
+* Beni Eugster (Swisscom)
+* Ionut Mihalcea (Arm)
+* Vijay Nayani (Huawei)
+* Xinxin Fan (IoTeX)
+* Marc Meunier (Arm)
+* Sampo Sovio (Huawei)
+
+#### Also Participating
+* Kurt Taylor (Linux Foundation)
+* Helen Lau (Linux Foundation)        
 
 ### Meeting minute approvals
 
 #### Approved by voting members
 
-<!--
-**RESOLVED:** That the minutes of the [March 10, 2022](../2022-03-10/CCC_TAC_Minutes-2022-03-10.md) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.-->
+**Resolved:** That the minutes of the [March 10, 2022](../2022-03-10/TAC_Minutes-2022-03-10.pdf) meeting of the Technical Advisory Council meeting of the Confidential Computing Consortium as distributed to the members of the TAC in advance of this meeting are hereby adopted and approved.
+
+ * Quorum reached, Minutes of March 10, 2022 were approved
 
 #### Pending approval
+ * None
 
-* [March 10, 2022](../2022-03-10/CCC_TAC_Minutes-2022-03-10.md)
+### TAC Tech Talks
+ *  Mark Novak - CC Governance Framework
+ 	*  Mark N, Thomas F, Naveen C, Steve VL, and Eric N expressed interest in collaborating on the governance topic, SIG formation possible
+ *  Eric Voit, Penglin Yang - Networking Confidential Computing
 
-### Open Action Items
 
-1. **[Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+### Open Action Items
 
-1. **[Mike, Eric V, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
+1. **[Mike, Eric, Stephen, Zongmin]** Recommend diversity and inclusion training to their projects and report back to the TAC on whether the maintainers or the contributors will be taking it and when the expected completion date is.
```

Comment:
Revert.  We have two Eric's (Eric N and Eric V) that are regular attendees so need to make sure the minutes accurately reflect which one in minutes.  (Line 103 still has the V so is fine)